www_src/pages/section-2/section-2.jsx

```diff
@@ -170,19 +212,19 @@ var Grid = React.createClass({
     this.gridWidth = undefined;
     this.gridHeight = undefined;
 
+    this.slotWidth = undefined;
```

Comment:
Is it redeclaring them or just re-assigning them? Is there a perf benefit to moving them into `componentWillMount`?

I'm kind of ambivalent on choosing one placement or the other since they both have roughly equal advantages in my mind.